.gitignore

```diff
@@ -18,3 +18,5 @@
 
 #Ignore images uploaded by paperclip
 public/system
+
+**/.DS_Store
```

Comment:
You should go put this in your global git ignore file on your machine so you don't have to do this all the time.